[BE] Rename DTOs in the API documentation

MihaiDruta opened the pull request:
[BE] Rename DTOs in the API documentation
https://trello.com/c/zr663EBb/43-be-rename-dtos-in-the-api-documentation

Renamed all models in documentation.

Commit:
[BE] Rename DTOs in the API documentation
https://trello.com/c/zr663EBb/43-be-rename-dtos-in-the-api-documentation

Renamed all models in documentation.

SpartakusMd commented on config/packages/nelmio_api_doc.yaml:
```diff
@@ -1,5 +1,30 @@
 nelmio_api_doc:
+  models:
+    names:
+      - { alias: CompleteActivity, type: App\Entity\Activity }
```
Just `Activity`, as it is the base one with most of the fields.

SpartakusMd commented on config/packages/nelmio_api_doc.yaml:
```diff
@@ -1,5 +1,30 @@
+      - { alias: ActivityDetails, type: App\Entity\Activity, groups: [ActivityDetails] }
+      - { alias: ActivityEdit, type: App\Entity\Activity, groups: [ActivityEdit] }
+      - { alias: ActivityCreate, type: App\Entity\Activity, groups: [ActivityCreate] }
+      - { alias: CompleteUser, type: App\Entity\User }
```
`User`

SpartakusMd commented on config/packages/nelmio_api_doc.yaml:
```diff
@@ -1,5 +1,30 @@
+      - { alias: UserCreate, type: App\DTO\UserDTO, groups: [UserCreate] }
+      - { alias: UserEdit, type: App\DTO\UserDTO, groups: [UserEdit] }
+      - { alias: UserPasswordEdit, type: App\DTO\UserDTO, groups: [PasswordEdit] }
+      - { alias: CompleteTechnology, type: App\Entity\Technology }
```
`Technology`

SpartakusMd commented on config/packages/nelmio_api_doc.yaml:
```diff
@@ -1,5 +1,30 @@
+      - { alias: TechnologyForActivityEdit, type: App\Entity\Technology, groups: [ActivityEdit] }
+      - { alias: TechnologyForUserDetail, type: App\Entity\Technology, groups: [UserDetail] }
+      - { alias: TechnologyForUserEdit, type: App\DTO\TechnologyDTO, groups: [UserEdit] }
+      - { alias: CompleteActivityType, type: App\Entity\ActivityType }
```
`ActivityType`

Commit:
[BE] Rename DTOs in the API documentation
https://trello.com/c/zr663EBb/43-be-rename-dtos-in-the-api-documentation

Renamed all models in documentation.
